Havit.Blazor.Components.Web.Bootstrap/Grids/HxPager.razor.cs

```diff
@@ -118,6 +119,16 @@ static HxPager()
 	[Parameter] public string CssClass { get; set; }
 	protected string CssClassEffective => CssClass ?? GetSettings()?.CssClass ?? GetDefaults().CssClass;
 
+	protected override void OnParametersSet()
+	{
+		if (TotalPages < 1)
+		{
+			throw new InvalidOperationException($"{nameof(TotalPages)} has to be greater than or equal to 1.");
+		}
+
+		base.OnParametersSet();
```

Comment:
(Blazor documentation suggests not to call empty `base` lifecycle methods when you inherit directly from `ComponentBase`.)